lib/generators/doorkeeper/templates/migration.rb

```diff
@@ -25,8 +26,8 @@ def change
     add_index :oauth_access_grants, :token, unique: true
 
     create_table :oauth_access_tokens do |t|
-      t.integer  :resource_owner_id
-      t.integer  :application_id
+      t.integer    :resource_owner_id, foreign_key: true, null: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.